fixtures/pnpm/package.json

```diff
@@ -6,11 +6,11 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "eslint": "^7.2.0",
+    "eslint": "\"^5.16.0 || ^6.8.0 || ^7.2.0\"",
     "eslint-config-airbnb": "18.2.1",
     "eslint-plugin-import": "^2.22.1",
     "eslint-plugin-jsx-a11y": "^6.4.1",
     "eslint-plugin-react": "^7.21.5",
-    "eslint-plugin-react-hooks": "^1.7.0"
+    "eslint-plugin-react-hooks": "\"^4 || ^3 || ^2.3.0 || ^1.7.0\""
```

Comment:
why are the extra quotes here?

Reply:
Good catch, looks like the range is extracted with quotes already, I simplified the code further but if someone could perform some manual QA, or tell me how I can test this more extensively that'd be great